cfp.html

```diff
@@ -336,12 +336,9 @@ <h3 class="front-button"><a href="tickets.html#ticket-types">BUY A TICKET</a></h
         <p>Proposal decisions will be sent no later than <strong>March 9th, 2015</strong>.</p>
 
         <h2 id="what-is-net-fringe"><span>WHAT IS .NET FRINGE?</span></h2>
-        <p><strong>.NET Fringe</strong> is a community of developers working on the edges of technology, creating new and interesting open source projects. Our conferences create a time and a place for this community to come together, share, discuss, and work together to build the next generation of amazing .NET software.</p>
-
-        <p>We think the .NET platform is more interesting today than it ever has been. The war is over and the notion that Microsoft and Open Source are at odds with one-another is antiquated. ALT.NET, which was founded on the idea of that conflict, is no longer relevant. <strong>.NET Fringe</strong> is a new community based on trust and collaboration, instead of conflict.</p>
-
-        <p>Come be a part of this fantastic gathering of open-source .NET developers!</p>
+         <p>We think the .NET platform is more interesting today than it ever has been! The war is over and the notion that Microsoft and Open Source are at odds with one-another is antiquated. <strong>.NET Fringe</strong> is a new event focused on the growth of OSS in .NET. The OSS community has been collaborating and creating new, really exciting, interesting, and innovative projects which are changing the .NET landscape. The community is defining the future!  </p>
```

Comment:
That was in the original verbiage but I think this referring to the
anti-Linux / OSS is evil days ;-)
On Wed, Feb 25, 2015 at 8:36 PM Phil Haack <notifications@github.com> wrote:

> In cfp.html
> <https://github.com/dotnetfringe/dotnetfringe.github.io/pull/2#discussion_r25404333>
> :
>
> > @@ -336,12 +336,9 @@ <h3 class="front-button"><a href="tickets.html#ticket-types">BUY A TICKET</a></h
> >          <p>Proposal decisions will be sent no later than <strong>March 9th, 2015</strong>.</p>
> >
> >          <h2 id="what-is-net-fringe"><span>WHAT IS .NET FRINGE?</span></h2>
> > -        <p><strong>.NET Fringe</strong> is a community of developers working on the edges of technology, creating new and interesting open source projects. Our conferences create a time and a place for this community to come together, share, discuss, and work together to build the next generation of amazing .NET software.</p>
> > -
> > -        <p>We think the .NET platform is more interesting today than it ever has been. The war is over and the notion that Microsoft and Open Source are at odds with one-another is antiquated. ALT.NET, which was founded on the idea of that conflict, is no longer relevant. <strong>.NET Fringe</strong> is a new community based on trust and collaboration, instead of conflict.</p>
> > -
> > -        <p>Come be a part of this fantastic gathering of open-source .NET developers!</p>
> > +         <p>We think the .NET platform is more interesting today than it ever has been! The war is over and the notion that Microsoft and Open Source are at odds with one-another is antiquated. <strong>.NET Fringe</strong> is a new event focused on the growth of OSS in .NET. The OSS community has been collaborating and creating new, really exciting, interesting, and innovative projects which are changing the .NET landscape. The community is defining the future!  </p>
>
> "The war is over..." What war?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dotnetfringe/dotnetfringe.github.io/pull/2/files#r25404333>
> .
>